pages/[user].tsx

```diff
@@ -55,7 +56,8 @@ export default function User(props: inferSSRProps<typeof getServerSideProps>) {
                 alt={nameOrUsername}
               />
               <h1 className="mb-1 text-3xl font-bold font-cal text-neutral-900 dark:text-white">
-                {nameOrUsername}
+                {nameOrUsername}{" "}
```

Comment:
Very small nitpick, but I'd render the space conditionally if verified is true, otherwise we have a random space.

Can be fixed by adding this to the line below:
```
verified && " " + <Verified...
```